python3/CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # IBM Functions Python 3 Runtime Container
 
+## 1.4.0
+Changes:
+- updated httplib2 (0.11.3)
+- updated kafka_python (1.4.2)
```

Comment:
Could you add the previous version and the new version for each package updated?
Here is an example on how we are doing it for nodejs
https://github.com/ibm-functions/runtime-nodejs/blob/master/nodejs8/CHANGELOG.md#160

Changes:
  - update NodeJS Runtime from `8.10.0` to `8.11.0`
  - update btoa from `1.1.2` to `1.2.1`
  - update commander from `2.15.0` to `2.15.1`
  - update elasticsearch from `14.1.0` to `14.2.2`